app/CMakeLists.txt

```diff
@@ -257,7 +257,12 @@ if (ANDROID)
     ${CMAKE_CURRENT_SOURCE_DIR}/android/AndroidManifest.xml @ONLY
   )
   set_target_properties(
-    Input PROPERTIES QT_ANDROID_PACKAGE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/android
+    Input
+    PROPERTIES QT_ANDROID_PACKAGE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/android
+               QT_ANDROID_TARGET_SDK_VERSION 31
```

Comment:
Should be expanded from variable , not hardcoded here (both 31 and 24) https://github.com/MerginMaps/input/blob/511fd2a7abca4554edc8e3f9abc6169c0971df4d/CMakeLists.txt#L21

Reply:
in any case it would be good to clean up those variables - I guess the issue is that those variables were defined, but not used anywhere, and the `QT_` prefix looked like it is something we set for Qt framework. Maybe in the main cmakelists file we could change the variable names to e.g. `INPUT_ANDROID_TARGET_SDK_VERSION` just to make it clear it is "our" variable

Reply:
addressed